src/components/NavBar.module.css

```diff
@@ -1,29 +1,33 @@
 :root {
   --main-color: #062d1d;
-  --secondary-color: #E3AA4D;
+  --secondary-color: #e3aa4d;
   --support-color: #ffffff;
 }
 
 .background {
   background-color: var(--main-color);
 }
 
-.navLink {  
-  color: var(--support-color)!important;
+.navLink {
+  color: var(--support-color) !important;
+  font-family: "Merriweather", serif;
 }
 
-.navLink:hover {  
+.navLink:hover {
   color: var(--secondary-color) !important;
+  font-family: "Merriweather", serif;
   font-size: 1.05rem;
   transition: 0.2s;
 }
 
 .endText {
+  font-family: "Merriweather", serif;
   color: var(--secondary-color) !important;
 }
 
-.endText:hover  {
-  color: var(--support-color)!important;
+.endText:hover {
+  font-family: "Merriweather", serif;
+  color: var(--support-color) !important;
   font-size: 1.05rem;
   transition: 0.2s;
-}
\ No newline at end of file
+}
```

Comment:
Same here. You don't need the extra font-family change.

Reply:
done